about.html

```diff
@@ -0,0 +1,157 @@
+        <div class="describe_container">
+          <h2 class="describe_container-title">
+            Gyi Gyi Paw Food and Drink Restaurant
+          </h2>
+          <p class="describe_container-body">
+            "Step into an oasis of elegance and charm as you enter our thoughtfully
+             designed space, adorned with stylish decor and soft lighting. Whether 
+             you're celebrating a special occasion or enjoying a casual evening out, 
+             our warm and inviting atmosphere sets the perfect stage for an unforgettable 
+             dining experience."
+          </p>
+          <div class="describe_container-container">
+            <h3 class="about-logo">GGP Food&Drink</h3>
+            <div class="describe_container-container-details">
+              <h2 class="describe_container-container-details-text">Daily-Open</h2>
+              <p class="describe_container-container-details-body">
+                9:00AM ~ 1:00PM
+              </p>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="previous">
+        <div class="previous_container">
+          <h4 class="previous_container-header">
+            See The Activity and Event In Restaurant
+          </h4>
+          <p class="previous_container-body">
+            Take a look at the last two event which took place in 
+            our Restaurant.
+          </p>
+          <div class="previous_container-images">
+            <div class="previous_container-images-card">
+              <img src="./images/event-1.jpg" alt="event 1" class="previous_container-images-card-image"/>
+              <div class="previous_container-images-card-details">
+                <div class="previous-text">
+                  <h2 class="previous__ontainer-images-card-details-header">
+                    2023
+                  </h2>
+                  <p class="previous_container-images-card-details-body">
+                    GGP Food and Drink Restaurant
+                  </p>
+                </div>
+              </div>
+            </div>
+
+            <div class="previous_container-images-card">
+              <img src="./images/event-2.jpg" alt="event 2" class="previous_container-images-card-image"/>
+              <div class="previous_container-images-card-details">
+                <div class="previous-text">
+                  <h2 class="previous_container-images-card-details-header">
+                    2023
+                  </h2>
+                  <p class="previous_container-images-card-details-body">
+                    GGP Food and Drink Restaurant
+                  </p>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+    </main>
+
+    <footer class="about-footer">
+      <div class="footer_container">
+        <h3 class="logo">GGP Food&Drink</h3>
+        <p class="reserve">All rights reserved.</p>
+      </div>
+    </footer>
```

Comment:
- [ ]  It will be better if you stick to the color of the Behance design as you see you used different colors in the text of the footer, so please try to stick to the color to make it nice like the Behance 💯👌
- [ ] Also, consider increasing the padding of the footer to be like the Behance as you see in your case the padding of the footer is less than the Behance, so let's try to increase the padding to make it nice 💯🚀

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Original design | Your design| 
| --- | --- |
|   ![image](https://github.com/YoutLone/Capstone-Project/assets/83360636/f8a2fab1-498e-4817-992f-b40b29da19f8)  |   ![image](https://github.com/YoutLone/Capstone-Project/assets/83360636/fbc814a7-e031-4800-b0ad-cf9c151b5f2e)  |